lib/bolt/node.rb

```diff
@@ -27,11 +27,12 @@ def self.from_uri(uri_string, **kwargs)
                 **kwargs)
     end
 
+    def self.initialize_transport(_logger); end
```

Comment:
@adreyer I didn't change `Bolt::Node::Orch` to load its config and create the orch client, because I have no idea how to test those changes.

Reply:
it's currently just tested in the client library. I'm ok not moving it for now.